sw/airborne/firmwares/rover/main.c

```diff
@@ -0,0 +1,3 @@
+// Empty... rovers only use main_ap...
```

Comment:
why do you need this file ?

Reply:
If I don't add this main.c then I get this error when trying to compile with NPS target:

make[1]: *** No rule to make target '/home/jesus/workspace/repos/paparazzi/var/aircrafts/ST_Rover/nps/firmwares/rover/main.c', needed by '/home/jesus/workspace/repos/paparazzi/var/aircrafts/ST_Rover/nps/firmwares/rover/main.o'.  Stop.
make[1]: *** Waiting for unfinished jobs....
CC /home/jesus/workspace/repos/paparazzi/var/aircrafts/ST_Rover/nps/firmwares/rover/main_ap.o
make[1]: Leaving directory '/home/jesus/workspace/repos/paparazzi/sw/airborne'
make: *** [Makefile.ac:213: nps.compile] Error 2